src/adapter/src/coord/command_handler.rs

```diff
@@ -71,6 +73,10 @@ impl Coordinator {
             Command::CopyRows { tx, session, .. } => send(tx, session, e),
             Command::GetSystemVars { tx, session, .. } => send(tx, session, e),
             Command::SetSystemVars { tx, session, .. } => send(tx, session, e),
+            Command::AppendWebhook { tx, .. } => {
```

Comment:
Is this tested somewhere?

Reply:
It is not, AFAICT this method is only called if we fail RBAC, and there aren't any RBAC restrictions on appending to a webhook, so this code path isn't used